CLAUDE.md

```diff
@@ -19,9 +19,124 @@
 - **Type Check:** `yarn type-check`
```

Comment:
I'm a bit wary of longer instructions, but this is not based on any benchmarking or anything. I think we should try and find out. If we use it enough and get to the point where we want to know what works best, we can then make benchmarks.

Reply:
You're right I'll try distilling them a bit